### PR TITLE
CI: Apply Vox Pupuli best practices & Add Ruby 3.2 support

### PR DESCRIPTION
**Pull request by bastelfreak**
also contains:
* https://github.com/voxpupuli/beaker-module_install_helper/pull/37
* https://github.com/voxpupuli/beaker-module_install_helper/pull/38

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -44,7 +45,7 @@ jobs:
           ruby-version: ${{ matrix.ruby }}
           bundler-cache: true
       - name: spec tests
-        run: bundle exec rake
+        run: bundle exec rake spec
```
Comment (ekohl) [inline .github/workflows/test.yml:48]: I think this (rake -> rake spec) deserves its own commit. Perhaps even make it part of the best practices PR.

File: .rubocop_todo.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  Max: 127
+
+# Offense count: 10
+RSpec/AnyInstance:
+  Exclude:
+    - 'spec/unit/beaker/module_install_helper_spec.rb'
+
+# Offense count: 1
+# Configuration parameters: AllowedConstants.
+Style/Documentation:
```
Comment (ekohl) [inline .rubocop_todo.yml:31]: Should we just disable this cop?

File: .rubocop_todo.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+# `rubocop --auto-gen-config`
+# on 2023-03-24 14:42:11 UTC using RuboCop version 1.28.2.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
+Metrics/AbcSize:
```
Comment (ekohl) [inline .rubocop_todo.yml:11]: I'm tempted to disable the metrics cops.

Comment (bastelfreak) [reply .rubocop_todo.yml:11]: will do so in a follow up PR to introduce voxpupuli-rubocop

File: .rubocop_todo.yml (hunk 2 above)
Comment (bastelfreak) [reply .rubocop_todo.yml:31]: will do so in the voxpupuli-rubocop followup PR

**Review by jay7x — APPROVED** [review]
